### PR TITLE
Installation rules

### PR DESCRIPTION
**Pull request by imobachgs**
Integration test for [bsc#963137](https://bugzilla.suse.com/show_bug.cgi?id=963137).

These tests shouldn't be merged until the fix is merged in master.

### CODE REVIEW

**Comment by schubi2** [comment]
Great ! LGMT

**Comment by imobachgs** [comment]
Thanks! I'll merged it as soon as the fix is available in master (waiting for users' feedback).

**Comment by mvidner** [comment]
IIUC https://github.com/yast/yast-autoinstallation/pull/202 was the merge of the fix into master, right?

**Comment by imobachgs** [comment]
Yes, that's it. I've rebased this PR against `master` again. Please, could you take a look (well, only the changes file was modified).

Thanks.

File: package/aytests-tests.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Mar 18 09:02:17 UTC 2016 - igonzalezsosa@suse.com
+
+- Add a test to check if AutoYaST rules were correctly evaluated.
```
Comment (mvidner) [inline package/aytests-tests.changes]: Please mention (bsc#963137) also here

Comment (mvidner) [inline package/aytests-tests.changes]: Never mind, I did that.

**Comment by mvidner** [comment]
LGTM!

**Pull request merged**